webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -198,7 +198,7 @@ const WorkspaceDetails: React.FC = () => {
 
     if (canViewModelRegistry({ workspace })) {
       items.push({
-        children: <ModelRegistry workspace={workspace} />,
+        children: <ModelRegistry hideBreadcrumb={true} workspace={workspace} />,
```

Comment:
@EmilyBonar I added a note about why I added `hideBreadcrumb` in the commentary section of this PR. I am curious if this should be refactored a different way.

Reply:
It makes sense why you're handling it this way, though I'm not a huge fan of components being both top-level pages and embedded in tabs. I think this is good for the moment and we can think about this again later.

Reply:
We can fix it in this PR. I will make the refactor.